tsconfig-lax.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
```

Comment:
Is there a specific place that we came up with this list of options from?  If so, might be nice to document as a comment on the BUILD target.

Reply:
I left a short sentence on it but I don't think I can comment much on it more than I did.

Reply:
Ok, no problem.